openlibrary/templates/books/breadcrumb_select.html

```diff
@@ -42,27 +42,4 @@ <h2 class="breadcrumb-title">$breadcrumb_title</h2>
             </span>
         </span>
 
-        <script>
-            const crumbs = document.querySelectorAll('.crumb select')
-            const allowedKeys = new Set(['Tab', 'Enter', ' '])
-            const preventedKeys = new Set(['ArrowUp', 'ArrowDown'])
-
-            // watch crumbs for changes,
-            // ensures it's a full value change, not a user exploring options via keyboard
-            crumbs.forEach(nav => {
-                let ignoreChange = false
-
-                nav.addEventListener('change', e => {
-                    if (ignoreChange) return
-                    // it's actually changed!
-                    window.location = nav.value;
-                })
-
-            nav.addEventListener('keydown', ({ key }) => {
-                if (preventedKeys.has(key))
-                ignoreChange = true
-                else if (allowedKeys.has(key))
-                ignoreChange = false
-            })
-            })
-        </script>
+        <script src="breadcrumb_select.js"></script>
```

Comment:
It is okay to do this while you're testing, but our goal is to get rid of script tags in the html files. So you'll need to add something index.js like [here](https://github.com/internetarchive/openlibrary/pull/8275/files#diff-587fb865c85bb3325504c795a40db189d2e702d42d9d160304607e027c142ebdR514).

Please be sure to follow the steps in #4474 which has more info